osgar/tools/lidarview.py

```diff
@@ -105,6 +107,23 @@ def draw(foreground, pose, scan, poses=[], image=None, callback=None, acc_pts=No
             for a, b in zip(pts[:-1], pts[1:]):
                 pygame.draw.line(foreground, (200, 200, 0), a, b, 1)
 
+g_depth = None
+def get_image(data):
+    """Extract JPEG or RGBD depth image"""
+    # https://stackoverflow.com/questions/12569452/how-to-identify-numpy-types-in-python
+    if type(data).__module__ == np.__name__:
```

Comment:
What's wrong with `isinstance(data, np.ndarray)`? It is in the second answer on that SO page linked above and we already use it in serialize.